config/default.js

```diff
@@ -70,5 +70,15 @@ module.exports = {
       host: 'api.geocode.earth',
       api_key: process.env.PELIAS_API_KEY || null
     }
+  },
+  geoip: {
+    host: 'api.ipstack.com/check',
+    api_key: process.env.GEOIP_API_KEY || null,
+    protocol: 'http://'
+  },
+  redis: {
+    port: process.env.REDIS_PORT,
+    secret: process.env.REDIS_SECRET,
+    redis_to_go_url: process.env.REDISTOGO_URL || null
```

Comment:
I think we should decouple ourselves from any particular Redis service provider. (for example, in addition to RedisToGo, there's also [Heroku Redis](https://elements.heroku.com/addons/heroku-redis), which uses a [different (and much more generic) env variable](https://devcenter.heroku.com/articles/heroku-redis#redis-module)).

One way we can do this is name this config setting `redis.url` instead (much shorter) and point it to a generic `process.env.REDIS_URL`. Then, for the production service's configuration, in `config/production.js`, we can overwrite `redis.url` with the deployment environment's actual environment variable, pointing `redis.url` at `process.env.REDISTOGO_URL` there.

---

Side question: If Redis has passwords, should the property and env name be `redis.password` and `REDIS_PASSWORD` respectively? Secrets usually imply a secure interaction with an API, but Redis appears to use servers with username/passwords.

Reply:
`redis.password` should be fine. I actually just didn't know what to call it.

Reply:
Should I point `redis.url` to `process.env.REDISTOGO_URL` in `config/production.js` now or should we figure out which Redis service provider to use first? I actually haven't researched as much as I should about what is different between these Redis service providers.